### DISCUSSION TITLE
[2024년 05월 22일] 데일리 스크럼 - 랭크, 이미지 url 반환 API 점검

### DISCUSSION THREAD

**Discussion by Subak-Uncle** [cca-ffodregamdi/running-hi-back-v2#241] [dailyscrum]
### 기록

종완

### 📋 회의 내용

- 배포
  - 현재 도메인(runninghi.store) 연결까지는 완성
  - 헌데 80 -> 8080 포트포워딩 설정을 했음에도 포워딩이 안되고 있음
  - 방화벽 설정과  포워딩 설정은 잘 되어 있음을 확인. 원인 파악 중입니다.
- API 추가 고려 사항
  - `챌린지 페이지` 개설과 함께 `마이페이지` 에 상세 API 점검이 필요
  - `05월 27일(월)` 회의에서 함께 상의
- 순위 API
  - 동등한 순위일 때, 동일한 순위를 부여하고 그 다음 순위를 부여하도록 설계
  - `OrderBy` 보다 `Rank()` 함수를 이용하는 것이 동순위 처리가 가능하여 `Rank()`함수 적용.
- 북마크 개수 API
  - 게시물 조회 응답에 포함되어야 한다는 것을 확인.
    게시물 조회 API에서 함께 조회되도록 설정 필요.
  - 추가로 기본 API도 추가해둘 예정
- 게시물 기록 생성 시 버그 발생
  - 게시물 기록을 `gzip`형태로 반복 요청을 보냈을 때 이동 거리가 실제 거리 보다 더 측정되는 버그 발견.
  - 정인님이 확인 예정.
- POSTMAN 업데이트
  - post API 기록 생성
  - 순위 API

**Comment by numerical43** [comment]
깔끔하고 아름다운 문서네요

**Reply by Subak-Uncle** [reply]
껄껄껄 감사합니다:)
